go.mod

```diff
@@ -35,13 +35,15 @@ require (
 	github.com/subosito/gotenv v1.6.0 // indirect
 	github.com/tarantool/go-iproto v1.0.0 // indirect
 	github.com/vmihailenco/tagparser/v2 v2.0.0 // indirect
+	github.com/wadey/gocovmerge v0.0.0-20160331181800-b5bfa59ec0ad // indirect
```

Comment:
Are these changes in go.mod, go.sum files required?

Reply:
no; i fixed it